projects/wasmtime/differential_spec.options

```diff
@@ -0,0 +1,5 @@
+[asan]
```

Comment:
I'm not 100% sure this will be sufficient. I wonder if libFuzzer needs to be told not to detect leaks as well.

Reply:
Ah ok I wasn't sure either if this configuration would take hold. After some discussion with a Wasmtime contributor who knows more about OCaml though it looks like we may be able to remove this leak-detection-exemption -- https://github.com/google/oss-fuzz/pull/7361 -- by using an older version of OCaml without the leak.